.dockerfiles/entrypoint-dev-rails.sh

```diff
@@ -9,7 +9,9 @@ npm list &> /dev/null || npm ci
 # install python packages
 [ -f ./venv/bin/python3 ] || python3 -m venv ./venv
 ./venv/bin/python3 -m pip install --upgrade pip > /dev/null
-./venv/bin/python3 -m pip install -r requirements-jupyter.txt -r requirements-scanner.txt > /dev/null
+./venv/bin/python3 -m pip install -r requirements-jupyter.txt > /dev/null
+./venv/bin/python3 -m pip uninstall -y -r requirements-scanner.txt > /dev/null
```

Comment:
@anissingh Okay I looked into this a little bit and didn't find anything conclusive, but my hypothesis is the installation of a fresh commit hash will work if the package version is bumped. Feel free to make a PR on markus-exam-matcher to try it out.

That's a better fix than uninstalling here (and is more in line with where we'll eventually end up, which is installing from the official package repository PyPI rather than GitHub).

Reply:
@david-yz-liu Sounds good, just made the PR.

Reply:
@anissingh Okay I just merged it in, please give it a try!

Reply:
It works, thanks! One thing I should point out, however, is that before integrating this package into MarkUs, `requirements-scanner.txt` contained a particular distribution of `opencv-python-headless`. Installing this package does not remove `opencv-python-headless` from the installed libraries because we switched over to `opencv-python`. Besides the fact that installing an unused library is unnecessary, it appears that it could cause errors if the versions do not align, which I do not believe they do in our case (see [here](https://stackoverflow.com/a/74245841), [here](https://stackoverflow.com/a/70898025), and [here](https://stackoverflow.com/a/71112424)). Although it doesn't seem to be dangerous in our environment, it's something that should maybe be addressed nonetheless.